binding/dotnet/Porcupine/Porcupine.csproj

```diff
@@ -101,6 +119,22 @@ Porcupine is:
 			<Link>resources\keyword_files\raspberry-pi\%(Filename)%(Extension)</Link>
 			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
 		</Content>
+        <Content Include="..\..\..\resources\keyword_files\jetson\*">
```

Comment:
I know it's a pain, but now that we're touching this SDK, we should take the opportunity to restrict the built-in keywords to our standard set:
- "alexa"
- "americano"
- "blueberry"
- "bumblebee"
- "computer"
- "grapefruit"
- "grasshopper"
- "hey google"
- "hey siri"
- "jarvis"
- "ok google"
- "picovoice"
- "porcupine"
- "terminator"